fdbserver/DataDistributionQueue.actor.cpp

```diff
@@ -1514,13 +1854,23 @@ ACTOR Future<Void> BgDDMountainChopper(DDQueueData* self, int teamCollectionInde
 			state Future<Void> delayF = delay(rebalancePollingInterval, TaskPriority::DataDistributionLaunch);
 			if ((now() - lastRead) > SERVER_KNOBS->BG_REBALANCE_SWITCH_CHECK_INTERVAL) {
 				tr.setOption(FDBTransactionOptions::LOCK_AWARE);
+				tr.setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
```

Comment:
Yeah, actually `FDBTransactionOptions::READ_SYSTEM_KEYS` will do nothing when using a raw `Transaction`, though I feel it's more clear to say this transaction read system keys in case there are more restrictions on system keys in the future.